include/dca/phys/dca_loop/dca_loop.hpp

```diff
@@ -15,13 +15,13 @@
 
 #include <cmath>
 #include <iostream>
+#include <filesystem>
```

Comment:
We still need to support gcc 7.3.0 and filesystem is only partially available and for that compiler it needs
```c++
#include <experimental/filesystem>
```

Reply:
why do we need to support gcc 7? I thought we agreed DCA++ is a c++17 code.

Reply:
I agree but we're a bit stuck on this one. I was not aware that std::filesystem was an exception for gcc7 until now.